hack/metrics/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
```

Comment:
Out of curiosity, does the value of `config.yaml` get merged if the cluster already has something? Presumambly not, right?